src/components/Blockquote/__snapshots__/Blockquote.spec.tsx.snap

```diff
@@ -4,22 +4,22 @@ exports[`Blockquote should render with default styles 1`] = `
 .circuit-0 {
   font-weight: 400;
   margin-bottom: 16px;
-  font-size: 13px;
-  line-height: 20px;
+  font-size: 16px;
+  line-height: 24px;
```

Comment:
The default size of the Text component is `mega`, whereas the default size of the Blockquote used to be `kilo`. That's why the font-size has changed here. I'm okay to change the default size of the Blockquote (it would need to be marked as a breaking change), but then we also need to update it [here](https://github.com/sumup-oss/circuit-ui/pull/617/files#diff-a078094fcfe30d094fd665ca7f713666R42).

Reply:
Ah yep, I think it makes sense to make it consistent with `Text`. I'll update the default!

Reply:
Done in 2855fd6